### PR TITLE
#16 Create filters component layout

### PR DESCRIPTION
**Pull request by Art-mimiron**
#16

### CODE REVIEW

File: src/app/shell/main/filters-list/filters-list.component.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<div class="filter">
+    <h2 class="filter__header">Знайди найкращі гуртки у своєму місті!</h2>
+    <div class="filter__searchbar">
+        <input class="filter__searchbar-input" type="text"  placeholder="Баскетбол / Динамо / ЦТДЮ / Драма / JavaScript">
```
Comment (dmitryy90) [inline src/app/shell/main/filters-list/filters-list.component.html:4]: I can see that some of you use 2-space tab and some 4-space tab, please be consistent. I would suggest using 2 spaces

**Review by dmitryy90 — CHANGES_REQUESTED** [review]
we dont use npm, use yarn instead

File: src/app/shell/main/filters-list/filters-list.component.html (hunk 1 above)
Comment (Art-mimiron) [reply src/app/shell/main/filters-list/filters-list.component.html:4]: I've deleted package-lock, looks like I accidentally install it

**Comment by Art-mimiron** [comment]
> we dont use npm, use yarn instead

I've deleted package-lock, looks like I accidentally install it

**Review by dmitryy90 — APPROVED** [review]

**Pull request merged**